examples/src/main/scala/zio/config/examples/ListExample.scala

```diff
@@ -35,4 +34,40 @@ object ListExample extends App with EitherImpureOps {
         PgmConfig("something", List("australia", "canada", "usa"))
       )
   )
+
+  val propertyTree =
+    write(config, PgmConfig("something", List("australia", "canada", "usa")))
+
+  assert(
+    propertyTree ==
+      Right(
+        PropertyTree
+          .Record(
+            Map(
+              "xyz"     -> Leaf("something"),
+              "regions" -> PropertyTree.Sequence(List(Leaf("australia"), Leaf("canada"), Leaf("usa")))
+            )
+          )
+      )
+  )
+
+  println(propertyTree.map(_.toHocon))
+  // Right(SimpleConfigObject({"regions":["australia","canada","usa"],"xyz":"something"}))
+  println(propertyTree.map(_.toHocon.render()))
+
+//  Right({
+//    # hardcoded value
+//      "regions" : [
+//    # hardcoded value
+//      "australia",
+//    # hardcoded value
+//      "canada",
+//    # hardcoded value
+//      "usa"
+//    ],
+//    # hardcoded value
+//      "xyz" : "something"
```

Comment:
It's sort of baffling to see why that `hardcoded value` came in.
I tried it manually, and still the structure produces extraneous comments.

Reply:
However it doesn't affect result (meaning, we are able to read back)

Reply:
It comes only when rendering as string. No information about doc/comment is kept in the actual `SimpleConfigObject` structure